Providers/Modules/Plugins/ChangeTracking/plugin/changetracking_lib.rb

```diff
@@ -5,9 +5,18 @@
 require 'logger'
 
 require_relative 'oms_common'
+require_relative 'oms_configuration'
 
 class ChangeTracking
 
+    @@proxy_conf_path = '/etc/opt/microsoft/omsagent/conf/proxy.conf'
+    @@omsadmin_conf_path = '/etc/opt/microsoft/omsagent/conf/omsadmin.conf'
```

Comment:
Your values for `@@omsadmin_conf_path` and `@@cert_path` and `@@key_path` are also not multi-homing compatible. Keep in mind that this will have to change soon.

Have you tested this in a scenario where these locations will not exist? (example: onboard primary, onboard secondary, remove primary, DSC gets applied) Please make sure it fails cleanly.